Automattic/woocommerce-payments
# Try adding WooCommerce share key to Stripe onboarding

#### Changes proposed in this Pull Request

WooCommerce Core's Coming Soon mode makes it difficult for Stripe to approve the site because content is hidden. The site however is available when using a share key as a query parameter. This PR makes sure to send the url with that share key parameter to Stripe so their mechanism can properly assess the site.









#### Testing instructions





This PR needs to be tested at the same time as https://github.com/woocommerce/woocommerce/pull/47685. You can use [This link to set up the proper WooCommerce branch and this WooPayments branch](https://jurassic.ninja/create/?jetpack-beta&shortlived&nojetpack&woocommerce&woocommerce-payments-dev-tools&branches.woocommerce-payments=try/add-woo-share-key-stripe&branches.woocommerce=add/LYS-viz-options-to-wcpay)

* Complete the profiler with US address
* Turn on Coming Soon mode WooCommerce > Settings > Site Visibility
* While there, turn on the Share link and note the url
* Go through the WooPayments onboarding profile but before finishing open the network tab, preserve the log, and search for `self_assessment`.
* See the `url` query parameter reflect the share link url once landed on Stripe's KYC page

-------------------


**Post merge**
